var/spack/repos/builtin/packages/libstdcompat/package.py

```diff
@@ -90,7 +90,7 @@ def cmake_args(self):
         args = []
         cpp_compat = self.spec.variants["cpp_compat"].value
 
-        if "cpp_unstable" in self.spec:
+        if self.spec.satisfies("cpp_unstable"):
```

Comment:
```suggestion
        if self.spec.satisfies("+cpp_unstable"):
```

Reply:
This now seems to be `self.spec.satisfies("^cpp_unstable")`.

Reply:
What I mean here is that is shouldn't be `^cpp_unstable` since there is no such package. It should be `+cpp_unstable` so it refers to the variant. This still seems incorrect in be99139.